README.md

```diff
@@ -383,7 +383,16 @@ This is a collection of conference papers that interest me. The emphasis is focu
    - *Decomposes convolution into multiple dense and zero kernels for sparsity savings.*
 - **Block Convolution: Towards Memory-Efficient Inference of Large-Scale CNNs on FPGA.** (CAS)
 - **moDNN: Memory Optimal DNN Training on GPUs.** (University of Notre Dame, CAS)
-- HyperPower: Power and Memory-Constrained Hyper-Parameter Optimization for Neural Networks. (CMU, Google)
+- **HyperPower: Power and Memory-Constrained Hyper-Parameter Optimization for Neural Networks.** (CMU, Google)
+
+### 2018 DAC
```

Comment:
Many thanks, Ken. I intend to survey the DAC papers these days. Your contributions help us a lot. I find there are much more ml papers than the ones you list here. I guess you have picked the highest-quality papers from the proceedings. That's great!

Reply:
I might have left out some.. It would be great if other can keep adding interesting papers! :)